packages/datatrak-web/src/features/Survey/SurveyQuestion.tsx

```diff
@@ -24,51 +35,44 @@ const Placeholder = ({ name, type, id }) => {
   );
 };
 
-const InstructionQuestion = ({ text }) => {
-  return <QuestionPlaceholder>{text}</QuestionPlaceholder>;
-};
-
 export enum QUESTION_TYPES {
-  Binary = Placeholder,
-  Checkbox = Placeholder,
-  Date = Placeholder,
-  DateTime = Placeholder,
-  FreeText = TextField,
+  Binary = BinaryQuestion,
+  Checkbox = CheckboxQuestion,
+  Date = DateQuestion,
+  DateTime = DateTimeQuestion,
+  FreeText = TextQuestion,
   Geolocate = Placeholder,
   Autocomplete = Placeholder,
   Instruction = InstructionQuestion,
-  Number = TextField,
+  Number = TextQuestion,
   Photo = Placeholder,
-  Radio = Placeholder,
-  DaysSince = Placeholder,
-  MonthsSince = Placeholder,
-  YearsSince = Placeholder,
-  SubmissionDate = Placeholder,
-  DateOfData = Placeholder,
+  Radio = RadioQuestion,
+  SubmissionDate = DateQuestion,
+  DateOfData = DateQuestion,
   Entity = Placeholder,
   PrimaryEntity = Placeholder,
   CodeGenerator = Placeholder,
   Arithmetic = Placeholder,
   Condition = Placeholder,
 }
 
-interface SurveyQuestionProps {
-  type: keyof typeof QUESTION_TYPES;
-  name: string;
-  id: string;
-  register?: any;
-  label?: string;
-  code?: string;
-  text?: string;
-  options?: any;
-  config?: any;
-}
-export const SurveyQuestion = (props: SurveyQuestionProps) => {
-  const FieldComponent = QUESTION_TYPES[props.type];
+export const SurveyQuestion = ({ type, name, ...props }: SurveyQuestionFieldProps) => {
```

Comment:
I'm on the fence about whether it's worth typing out these props as well but since this component is quite simple and they are more directly passed on then it's probably all good to just spread them as is.